### ISSUE TITLE
Running formatter via NPX fails because the script is interpreted as shell

### ISSUE THREAD

**Issue by alexandercampbell** [oakmac/standard-clojure-style-js#94]
I cannot run the formatter via `npx`. The script is interpreted as shell rather than JS.

```
~/tmp 0 λ npx @chrisoakman/standard-clojure-style@latest fix example.clj
/Users/alexander/.npm/_npx/eb186f45d353568b/node_modules/.bin/standard-clj: line 1: //: is a directory
/Users/alexander/.npm/_npx/eb186f45d353568b/node_modules/.bin/standard-clj: line 2: //: is a directory
/Users/alexander/.npm/_npx/eb186f45d353568b/node_modules/.bin/standard-clj: line 3: //: is a directory
/Users/alexander/.npm/_npx/eb186f45d353568b/node_modules/.bin/standard-clj: line 4: //: is a directory
/Users/alexander/.npm/_npx/eb186f45d353568b/node_modules/.bin/standard-clj: line 5: //: is a directory
/Users/alexander/.npm/_npx/eb186f45d353568b/node_modules/.bin/standard-clj: line 6: //: is a directory
/Users/alexander/.npm/_npx/eb186f45d353568b/node_modules/.bin/standard-clj: line 8: //: is a directory
Version: ImageMagick 7.1.1-38 Q16-HDRI aarch64 22398 https://imagemagick.org
Copyright: (C) 1999 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI Modules OpenMP(5.0)
Delegates (built-in): bzlib fontconfig freetype gslib heic jng jp2 jpeg jxl lcms lqr ltdl lzma openexr png ps raw tiff webp xml zlib zstd
Compiler: gcc (4.2)
Usage: import [options ...] [ file ]

-- snip --
```

I think the problem was caused by removing the shebang from `cli.mjs` in a3ed01beda6249dab1b0a540a3b4b23b7aa68a2e.

Platform:
- MacOS
- Darwin.local 22.5.0 Darwin Kernel Version 22.5.0
- zsh 5.9 (x86_64-apple-darwin22.0)

**Comment by alexandercampbell** [comment]
I've repro'd this issue with these tool versions:
- npx 10.8.3 (latest)
- node 22.9.0 (latest)

**Comment by oakmac** [comment]
Fixed with [PR-96](https://github.com/oakmac/standard-clojure-style-js/pull/96)

I removed the shebang line because I wanted the ability to run `cli.mjs` using bun without having to [specify `--bun`](https://bun.sh/docs/cli/run#bun).

**Comment by oakmac** [comment]
Fixed with `v0.4.1` :+1:
